### PR TITLE
Rely on GS to preview accent color

### PR DESCRIPTION
**Pull request by dsas**
#### Proposed Changes
   
Newsletter accent colours are implemented using Global Styles. They are
previewed using an accent color specific parameter and serverside code
(D89679-code). PR #70834 introduces a more generic method of previewing,
which previews all of the global styles for a site.
    
This change removes the accent colour specific preview so we can just
rely on the generic global styles preview implementation.

Note that this PR is branched from PR #70834

Post-merge, I'll revert D89679-code

#### Testing Instructions

1. Open `/setup/newsletter/newsletterSetup` in a dev environment to get the correct feature flag (`limit-global-styles`).
2. Pick a "premium" color.
3. As soon as the site is created, add the `wpcom-limit-global-styles` sticker.
4. Get to the Launchpad's last step.
5. ➡️ Ensure the preview is using the "premium" color.

image

#### Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
  - No - it's an onboarding feature and can only be a simple site 
- [x] Have you checked for TypeScript, React or other console errors?
  - Didn't see anything new
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
  - There are no expensive computations
- [x] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
  - There are no new strings computations
- [x] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
  - Not relevant to Jetpack, it's a wpcom specific onboarding change.

Related to #

### CODE REVIEW

**Review by mmtr — APPROVED** [review]
Nice, this works well on my tests.

Let's wait for feedback from @Automattic/cylon before merging this. If there is any concern about removing the `preview_accent_color` param, we can land https://github.com/Automattic/wp-calypso/pull/70834 first and tackle the `preview_accent_color` param in a separate PR.

**Comment by jeyip** [comment]
cc @mattwiebe for having worked on D89679-code

**Comment by jeyip** [comment]
@mattwiebe says that this PR looks good 👍  p1670873047946649/1670870790.486109-slack-CHN6J22MP. @edanzer is double checking functionality

**Comment by edanzer** [comment]
Hi all! I'm testing this now as we well. I had also worked on much of the work on the site accent color and the launchpad web preview. We needed that accent color parameter to essentially bust caching of that value when loading the preview - specifically when updating that color after initial rendering of the preview. 

The issue would not be visible when first going through a flow and landing on Launchpad. Rather, it's becomes visible after you're on Launchpad, and then go to update/change the accent color. When you come back to Launchpad after doing so, the updated color was not rendered in the preview. 

I don't know quite enough about how some of global styles code is working to know if it will effectively still resolve/avoid that issue. I'll test now to confirm.

**Comment by edanzer** [comment]
@dsas @mmtr @jeyip - To follow up on my comment above, I've tested this and confirmed everything works as expected. In particular, if you go to Launchpad, click Personalize your site, change colors, and then return to Launchpad, the web preview loads correctly. I tried multiple times with different colors, including custom colors.

**Review by edanzer — APPROVED** [review]

**Comment by dsas** [comment]
Thanks @edanzer & @jeyip 👍. A whole load of caching fixes have been made to Core global styles in the last release, it's possible that they have fixed the colour updating issues you had before. I'll ship this tomorrow

**Pull request merged**
